app/controllers/planning_applications_controller.rb

```diff
@@ -155,6 +190,43 @@ def decision_notice
 
 private
 
+  def planning_application_params
+    permitted_keys = %i[address_1
+                        address_2
+                        application_type
+                        applicant_first_name
+                        applicant_last_name
+                        applicant_phone
+                        applicant_email
+                        agent_first_name
+                        agent_last_name
+                        agent_phone
+                        agent_email
+                        county
+                        created_at(3i)
+                        created_at(2i)
+                        created_at(1i)
```

Comment:
as we are manually setting the created_at here, I think we can remove them from this mass assignment?

Reply:
but then they won't be registered as allowed params when coming in from the form and updating as planning_application_params?

Reply:
yep I can't remove these or the update action stops working for the date

Reply:
You aren't setting these params in the mass assignment on line 46. You are manually setting it on line 47, so it doesn't need to be allowed via strong params

Reply:
I'm not setting them on the "create" action, but in the update action I don't explicitly assign those date attributes. I simply call
```
@planning_application.update(planning_application_params)
```
and the date gets updated. However, if I remove those 3 from strong params, the date update doesn't happen at all, because the created_at is no longer accepted as a param. I would need to instead of just calling
@planning_application.update(planning_application_params) also do some extra work on update to assign time attributes that now get updated just because of that params.

Reply:
if you give a branch a pull try deleting those params and then running spec/system/planning_applications/update_spec.rb

Reply:
Huh, that's weird. Why is it working via mass assignment on update, but not create? I can't look at it right now, but will try and take a look in a short while.

Reply:
it does also work on create, I just coped the time zone_parse method I'd seen as I assumed it was necessary.. but it wasn't! Deleted the extra method - my bad